spk/tt-rss/src/dsm-control.sh

```diff
@@ -18,21 +18,21 @@ HT_ACCESS_SECTION_DELIMITER="#Synology PHP"
 
 compute_generated_htaccess_section()
 {
-    local php_fpm="$1"
+    local php_profile_name="$1"
```

Comment:
php-fpm version based handlers was actually a backward compatible hack enforced by the webstation default configuration. 
For 7.3+ versions you actually need to rely on the handlers defined by the current php profile.